lib/macho/load_commands.rb

```diff
@@ -400,6 +400,24 @@ def initialize(view, cmd, cmdsize, segname, vmaddr, vmsize, fileoff,
       @flags = flags
     end
 
+
+    # All sections referenced within this segment.
+    # @return [Array<MachO::Section>] if the Mach-O is 32-bit
+    # @return [Array<MachO::Section64>] if the Mach-O is 64-bit
+    def sections
+      section_klass = case self
+      when MachO::SegmentCommand64
+        MachO::Section64
+      when MachO::SegmentCommand
+        MachO::Section
+      end
+
+      sections_bin = view.raw_data[offset + self.class.bytesize, nsects * section_klass.bytesize]
```

Comment:
> offset should be view.offset to avoid using the deprecated LoadCommand#offset method

Thanks for reminding me!



> One possible solution, since everything in here is about sections, would be to strip the prefix and use klass, bins, and bin.

:+1: